File: 20210719/거스름돈/거스름돈.py
returnCoin = 1000 - int(input())
coins = [500, 100, 50, 10, 5, 1]
count = 0

for coin in coins:
    if coin <= returnCoin:
        count += returnCoin // coin
        returnCoin = returnCoin % coin

print(count)

File: 20210719/거스름돈/거스름돈.js
function solution(n) {
  let returnCoin = 1000 - n;
  let coins = [500, 100, 50, 10, 5, 1];
  let count = 0;

  for (const coin of coins) {
    if (coin <= returnCoin) {
      count += paseInt(returnCoin / coin);
    }
  }
  print(count);
}
